scripts/trivy_vulnerabilities_data.sh

```
@@ -0,0 +1,218 @@
+
+# Function to install Trivy with retry logic
+install_trivy_with_retry() {
+    local count=0
+    local success=false
+
+    while [[ $count -lt $MAX_RETRIES ]]; do
+        echo "Attempting to install Trivy (attempt $((count + 1)))..."
+        
+        # Fetch the latest release dynamically instead of hardcoding
+        TRIVY_VERSION=$(curl -s https://api.github.com/repos/aquasecurity/trivy/releases/latest | grep '"tag_name"' | sed -E 's/.*"v([^"]+)".*/\1/')
+        TRIVY_URL="https://github.com/aquasecurity/trivy/releases/download/v$TRIVY_VERSION/trivy_"$TRIVY_VERSION"_Linux-64bit.tar.gz"
```

Shellcheck

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)